### ISSUE TITLE
Deeplinks With Tabs Widget

### ISSUE THREAD

**Issue by bhernandezbkj** [elementor/elementor#8844]
- [ x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

It would be a great idea to allow deep linking with the tabs widget, to allow bookmarking and other options. Here's an example for a website we added this to, in this case using accordions (with jQuery): https://www.greensalem.com/city-of-salem-recycling-info/a-z-guide/#books. Here's a general outline on how we coded this (just an example):

**After every page request:**
-loop through every item entry and apply an id to each based on the value
-In the loop mentioned above, add an href with the new id deeplinked {ex: url.com#newid}
-we get the hash of url using location.hash (if there is one)
-add a class of _active_  to the item entry with that id
-do some calculations to make sure the item entry is visible

**After a click**

-change url to corresponding id
-add class _active_ to item entry clicked on (remove it if item already has class _active_)
-make sure item entry is visible on the screen

This was just a general outline on how we handled this. It seems something like this could work for the tabs (a few classes like _active_ get added after a click)
![biotabs](https://user-images.githubusercontent.com/54150288/63118063-18345700-bf6b-11e9-8c1b-402dbb7258a3.JPG)
.

**What I expect to happen:**
Basically, have the ability to add an id to each tab in the elementor widget (in the same area where you add the actual tab value and data). Then I would like to bookmark www.example.com/#tabid (which would change according to tab clicked on) on my browser . When I revisit that link, the corresponding tab should open.

**Alternatives:**
As long has the tab bookmarked opens, anything works.

**Comment by crispee2000** [comment]
Four years later and this is still not possible?
